package.json

```diff
@@ -55,67 +47,55 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "^7.17.8",
-    "@babel/preset-env": "^7.16.11",
-    "@babel/preset-react": "^7.16.7",
-    "@babel/preset-typescript": "^7.16.7",
-    "@swc/core": "^1.2.196",
```

Comment:
My understanding is that they were recommending to install @swc/core (see their doc: https://swc.rs/docs/usage/jest)
But if that wotks without the entry in package.json I'm fine with that too, so that's up to you.